### PR TITLE
🚧 Allow enable trial

### PR DESCRIPTION
**Pull request by GusBaamonde**
🚧 WIP - DO NOT MERGE WITHOUT BACKEND

- Add a new scenario to enable the test
- allow enable trial free
PR realated with:  https://github.com/FromDoppler/Doppler-UI-System/pull/96

datahub-messages

### CODE REVIEW

**Comment by andresmoschini** [comment]
@carodipietro @santiagopaolucci could you take a look?

File: src/i18n/es.json [hunk 1]
```diff
@@ -45,6 +45,9 @@
     "you_want_create_account": "¿Aún no tienes una cuenta?"
   },
   "reports": {
+    "allow_enable_trial_HTML": "<p>Envía Emails automáticos y 100% personalizados a quienes visiten una página específica de tu sitio web o tienda online. Por ejemplo, puedes enviar un descuento a los usuarios que visitaron la página de un producto en particular para hacer que regresen a comprarlo.</p><p><strong>¿Cuándo finaliza el periodo de prueba?</strong><br />Nuestro objetivo es que puedas ver de qué forma utilizar esta funcionalidad en tu negocio y comprobar sus beneficios, por lo que por el momento podrás utilizarla sin limitaciones.</p><p><strong>¿Qué sucederá con las Automations creadas cuando finalice el periodo de prueba?</strong><br />Permanecerán inactivas, y no podrás editarlas hasta que contrates el módulo especial que incluye este servicio.</p>",
```
Comment (andresmoschini) [inline src/i18n/es.json]: What do you think about using headers (h1, h2, h3, etc) in place of strong and brs?

File: src/i18n/en.json [hunk 2]
```diff
@@ -45,6 +45,9 @@
     "you_want_create_account": "Do not you have an account yet?"
   },
   "reports": {
+    "allow_enable_trial_HTML": "<p>Target users based on their behavior on your website or E-commerce. Thanks to this feature you can send automatic and 100% personalized Emails to the users who visit a specific page. For example, you can send a discount to users who view a particular product page.</p><p><strong>When does the free trial end?</strong><br />Our goal is that you can see how to use this feature in your business and check its benefits, so for the moment you can use it with no restrictions.</p><p><strong>What will happen to the Automations created when the free trial ends?</strong><br />They'll remain inactive, and you won't be able to edit them until you hire the special module which includes this service.</p>",
+    "allow_enable_trial_button_HTML": "Activate free trial",
```
Comment (andresmoschini) [inline src/i18n/en.json]: In my opinion, in this case, we should not use HTML.

**Comment by andresmoschini** [comment]
Great work @GusBaamonde, Thursday morning we will connect it with the backend and merge it.

**Comment by GusBaamonde** [comment]
Closed in favor of https://github.com/FromDoppler/doppler-webapp/pull/159

**Pull request closed without merging**